src/Evolution/Systems/Burgers/Subcell/GhostData.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "Domain/Tags.hpp"
+#include "Evolution/DgSubcell/Tags/Mesh.hpp"
+#include "Evolution/Systems/Burgers/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <size_t Dim>
+class Mesh;
+template <typename T>
+class Variables;
+namespace Tags {
+template <typename TagsList>
+struct Variables;
+}  // namespace Tags
+/// \endcond
+
+namespace Burgers::subcell {
+/*!
+ * \brief Returns \f$U\f$ on the subcells so it can be used for reconstruction.
+ *
+ * This mutator is passed to
+ * `evolution::dg::subcell::Actions::SendDataForReconstruction`.
+ *
+ * \note Called only by FD-solving elements.
```

Comment:
I don't find the phrasing `FD-solving elements` particularly easy to understand. What do you think of `Only called on elements using FD.`?